### DISCUSSION TITLE
No Sound on my Chromebook

### DISCUSSION THREAD

**Discussion by tsarmark** [sebanc/brunch#1425] [General]
Hi guys, I have a little query. What happens is that I have an hp Chromebook 11 g4, the problem arises when I reinstall ChromeOS on my device since I have no audio, it is that I had installed Windows, neither in speakers nor in headphones. Does anyone know any solution? Thanks in advance

**Comment by sebanc** [comment]
Is the device board "kip" ?

**Reply by arcangelgabriel080293** [reply]
No. I use remmus

**Reply by sebanc** [reply]
Are you the same user ? If so, what is your chromebook original board ? (not the brunch suggested recovery)

**Reply by tsarmark** [reply]
Hi Sebanc, in fact the user above is the owner of the Chromebook, I'm just helping him. How would I know the type of keyboard? Thanks for the help

**Reply by arcangelgabriel080293** [reply]
I use the latinoamericano board

**Reply by sebanc** [reply]
Open the brunch configuration menu and choose "chromebook_audio" option and kernel chromebook 4.4

**Reply by arcangelgabriel080293** [reply]
Ok I understand

**Reply by tejasraman** [reply]
Board name is Kip. You need to choose ChromeOS(settings) at the boot menu and add the chromebook_audio option and use kernel 5.4-chromebook

**Reply by arcangelgabriel080293** [reply]
Tks the problem is fixed whit the change the kernel 4.4
